drivers/dai/intel/ssp/dai-params-intel-ipc4.h

```diff
@@ -223,17 +226,35 @@ struct dai_intel_ipc4_gateway_config_blob {
 /* i2s Configuration BLOB building blocks */
 
 /* i2s registers for i2s Configuration */
+#ifdef CONFIG_SOC_INTEL_ACE30_PTL
+struct ssp_rx_dir {
+	/* Specifies which time slots the DMA FIFO will receive from the SSP Interface*/
+	uint64_t ssmidytsa;
+} __packed;
```

Comment:
I read the "Attributes" gcc documentation carefully again and I can't see what difference this `__packed` attribute can possibly make. `__packed` removes internal padding and misaligns fields but there is no internal padding to remove when there is a single field.

More generally speaking, what is the point of a `struct` with a single field? I mean, why is `rx_dir` simply not just an array of uint64_t?

Reply:
Fixed